boards/FSM/Src/stm32f3xx_hal_callbacks.c

```diff
@@ -1,8 +1,10 @@
 #include <stm32f3xx_hal.h>
 
 #include "Io_FlowMeters.h"
+#include "Io_WheelSpeedSensors.h"
 
 void HAL_TIM_IC_CaptureCallback(TIM_HandleTypeDef *htim)
 {
     Io_FlowMeters_InputCaptureCallback(htim);
+    Io_WheelSpeedSensors_InputCaptureCallback(htim);
```

Comment:
Slightly out of scope but can we move this to `Src/Io/Io_InputCaptureCallback.c`. We should keep our code under `Io/` or `App` so it's more obvious what's written by us

Reply:
yeah that would make sense. done.